Mathlib/Tactic/Linter/DocString.lean

```diff
@@ -24,6 +24,25 @@ register_option linter.style.docString : Bool := {
   descr := "enable the style.docString linter"
 }
 
+/--
+Extract all `declModifiers` from the input syntax. We later extract the `docstring` from it,
+but we avoid extracting directly the `docComment` node, to skip `#adaptation_note`s.
```

Comment:
Can you add a test for #adaptation_notes's?

Reply:
The test file already contains some `#adaptation_note`s: did you have different tests in mind?

Reply:
No - confirming that this already exists suffices!